ui/page/components/account_selector.go

```diff
@@ -38,10 +39,14 @@ type AccountSelector struct {
 	changed            bool
 }
 
-func NewAccountSelector(l *load.Load) *AccountSelector {
+// NewAccountSelector opens up a modal to select the desired account. If a
+// nil value is passed for selectedWallet, then accounts for all wallets
+// are shown, otherwise only accounts for the selectedWallet is shown.
+func NewAccountSelector(l *load.Load, selectedWallet *dcrlibwallet.Wallet) *AccountSelector {
```

Comment:
I have my reservation on this approach though.

Reply:
alright, please make it known, so we can discuss

Reply:
So looking at the issue statement, the selected wallets come with 2 accounts (Account-1 and account-2) why have a dropdown when we can simply notify the users that we have spotted these 2 account, and if they will like to use it to set up the privacy or will they like set it up automatically.

Although i tried to test this nut after import, my mixer accounts were not imported.

Reply:
`(Account-1 and account-2) ` this is not a default name for privacy accounts after import, it can as well be account 7 or account 5, depending on the accounts the user has, so we won't be able to determine that way